tests/management/test_access_key.py

```diff
@@ -185,6 +187,7 @@ def test_update(self):
                 client.mgmt.access_key.update,
                 "key-id",
                 "new-name",
+                "new description",
```

Comment:
If you make `description` have a default value for update, please also test the empty description is sent when no value is set for it